examples/sdk/reactNative/android/app/src/main/java/com/reactnative/BacktraceDemoPackage.java

```diff
@@ -1,29 +0,0 @@
-package com.reactnative;
-
-import com.facebook.react.ReactPackage;
-import com.facebook.react.bridge.NativeModule;
-import com.facebook.react.bridge.ReactApplicationContext;
-import com.facebook.react.uimanager.ViewManager;
-
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-
-public class BacktraceDemoPackage implements ReactPackage {
-
-   @Override
-   public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
-       return Collections.emptyList();
-   }
-
-   @Override
-   public List<NativeModule> createNativeModules(
-           ReactApplicationContext reactContext) {
-       List<NativeModule> modules = new ArrayList<>();
-
-       modules.add(new ErrorGenerator(reactContext));
```

Comment:
where is the ErrorGenerator module added in the new react native demo?

Reply:
Restored.